homeassistant/components/plugwise/__init__.py

```diff
@@ -159,3 +204,36 @@ def _process_data(self):
     async def async_update(self):
         """Update the entity."""
         await self._coordinator.async_request_refresh()
+
+
+class SmileSensor(SmileGateway):
```

Comment:
If this class is only used by `sensor.py`, it probably makes sense to keep it in there

Reply:
We should be using it for `binary_sensor` as well when thats due.

Reply:
Between @bouwew and me we moved it to `sensor` and will pick it up from there once we get to `binary_sensor`, in our custom_component already preplanning the work.